### ISSUE TITLE
score

### ISSUE THREAD

**Issue by mytk2012** [kemaloksuz/aLRPLoss#3]
Appreciate for your work!Can you tell me how do you calculate the score?I have read your paper,so much mass,but it refers to more loss,but no expainations of score.

**Comment by kemaloksuz** [comment]
score is simply the unnormalized probability obtained from the classification network (i.e. without applying sigmoid to network prediction). We define it more clearly in Section S5 of Supp.Mat (Appendix E in arXiv version) under "description of inputs" as follows: "S is the raw output of the classification branch, namely logits". 

As for your question, we do not calculate scores, these are simply classification branch outputs.

**Comment by mytk2012** [comment]
amazing work!
